salt/modules/virt.py

```diff
@@ -2090,6 +2152,70 @@ def init(
     min_guarantee
         the guaranteed minimum memory allocation for the guest
 
+    .. _init-cputune-def:
+
+    .. rubric:: cputune parameters definition
+
+    The cputune parameters dictionary can contain the following properties:
+
+    vcpupin
+        The optional vcpupin element specifies which of host's physical CPUs the domain vCPU will be pinned to.
+
+    emulatorpin
+        The optional emulatorpin element specifies which of host physical CPUs the "emulator", a subset of a domain not
+        including vCPU or iothreads will be pinned to.
+
+    iothreadpin
+        The optional iothreadpin element specifies which of host physical CPUs the IOThreads will be pinned to.
```

Comment:
Mention here that the IOThread IDs to use in this value are from 1 to the number passed to `iothread` parameter.